miasm2/jitter/codegen.py

```diff
@@ -283,6 +283,8 @@ def traverse_expr_dst(self, expr, dst2index):
                         "0")
 
         else:
+            # Spot case like Cond?X:Y
+            assert -1 not in dst2index.values()
```

Comment:
```
Well, the value is erased in ...
```
The value of what?

I insist in order to see through your eyes.
In the code, only the default case will set an entry in `dst2index`.
The important part of the code is to keep the **value** and the **type** synchronized.

And I didn't see (for now :smile:) any case which case do this de-synchronization.
So for me two solutions here:
* either you remove the assert
* or you spend time to get an example which break the code